tests/functional/cephx_keyrotation/test_cephx_keyrotation_mismatch_alert.py

```
@@ -0,0 +1,1042 @@
+        "Set %s=%s on %s (previous=%s)",
+        name,
+        value_repr,
+        OCS_OPERATOR_DEPLOYMENT,
+        previous,
+    )
+    time.sleep(5)
+    _wait_for_ocs_operator_running(namespace)
+    return previous, existed
+
+
+def _remove_ocs_operator_env_var(namespace, name):
```

Ruff (0.15.21)

[warning] Missing return type annotation for private function `_remove_ocs_operator_env_var`

Add return type annotation: `None`

(ANN202)